python/perspective/perspective/table/table.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, data, limit=None, index=None):
         if self._is_arrow:
             _accessor = data
         else:
+            if isinstance(data, pandas.DataFrame):
+                if data[data.columns[0]].dtypes == "string":
```

Comment:
this checks if the first column is a string type, and if so converts all columns to object